optuna/visualization/_pareto_front.py

```diff
@@ -321,7 +321,7 @@ def _make_marker(
             "color": [t.number for t in trials],
             "colorscale": "Reds",
             "colorbar": {
-                "title": "#Best trials",
+                "title": "Best Trials Number",
```

Comment:
I agree with @hvy because the trial numbers and best trial numbers are not continuous, so the continuous color bar as in the master branch is a little bit confusing.

Reply:
Alright! Should I make the same changes ("Trial number" -> "Trial") in `plot_slice` and `plot_optimization_history` as well?

Reply:
Thanks, looks good.
As for the other plots, I think we can simplify them as you say as well, even though the optimization history is not continuous. What do you think @nzw0301 ?

Reply:
Sounds good. So in terms of `slice_plot`, we would use the same color among trials, not different colors depending on trial numbers, right?

Reply:
Ah, sorry I probably didn't make myself clear enough.
I meant still using different colors, just changing the labels, given this information is still useful for finding trends and seeing how the optimization progresses. Do you think it'll be confusing?

Reply:
Thank you for your clarification! I misunderstood the expected change. 

It makes sense.